src/conf_mode/vpn_l2tp.py

```diff
@@ -330,6 +330,14 @@ def verify(l2tp):
             if not radius['key']:
                 raise ConfigError(f"Missing RADIUS secret for server { radius['key'] }")
 
+        if l2tp['radius_dynamic_author']:
+            if not l2tp['radius_dynamic_author']['server']:
+                raise ConfigError("Missing ip-address for dae-server")
+            if not l2tp['radius_dynamic_author']['port']:
+                raise ConfigError("Missing port for dae-server")
```

Comment:
Is it possible to use a default port value?

Reply:
That would be nice

Reply:
Ok, I did changes but in the old style. The whole l2tp vpn will rewrite on 1.4 and then we can backport into 1.3